v2python/compile.py

```diff
@@ -15,7 +15,7 @@
 from triton.backends.compiler import GPUTarget
 
 KNOWN_TARGETS_MI = ['gfx90a', 'gfx942', 'gfx950']
-KNOWN_TARGETS_Navi = ['gfx1100', 'gfx1101', 'gfx1151', 'gfx1201']
+KNOWN_TARGETS_Navi = ['gfx1100', 'gfx1101', 'gfx1151', 'gfx1152', 'gfx1153', 'gfx1201']
```

Comment:
Files under v2python are deprecated and only kept for reference. They should not be updated any more.